### ISSUE TITLE
[Bug] `latents_attention_mask` not used

### ISSUE THREAD

**Issue by HaoyiZhu** [hao-ai-lab/FastVideo#168]
### Environment

/

### Describe the bug

Thanks for the amazing project!

In the data loader, latents are zero-padded and recorded by an attention mask. However, during training and model forward, it seems that the latent attention mask is not used. Is this a bug?

### Reproduction

/

**Comment by foreverpiano** [comment]
Thanks for pointing out this. latents_attention_mask is not used. I will remove it and refactor code soon.

**Comment by jzhang38** [comment]
We are going to add support for variable-length sequence packing. I intentionally retain the latent mask for this future feature.
